lib/app.dart

```diff
@@ -18,6 +18,7 @@ class _MyAppState extends State<MyApp> {
   Widget build(BuildContext context) {
     return OverlaySupport.global(
       child: MaterialApp(
+        debugShowCheckedModeBanner: false,
```

Comment:
I think we should keep this for debug mode, easier to distinguish between debug and release mode.